docs/install.md

```diff
@@ -1,38 +1,64 @@
 # Installation
 
-We recommend **the easy way** if you are running your own server.
+There are many options for installing Sandstorm with various trade-offs. Choose the one that is most comfortable for you.
 
-## Installing the Easy Way
+Sandstorm requires Linux x86_64, with kernel version 3.13 or later.
 
-*Prerequisite:* Linux x86_64, with kernel version 3.13 or later.
+## Option 1: HTTPS-verified install
 
-To install on your own Linux machine, see
-[https://install.sandstorm.io/](https://install.sandstorm.io) or run:
+The easiest way to install Sandstorm is by running:
```

Comment:
If this doc gets more pagerank than sandstorm.io/install/ then I'll be a little sad. Luckily (from some Googling) there's no serious risk of that right now.

Consider linking back to sandstorm.io/install/ here, but I'm not sure where. So maybe don't bother.